paymentsheet/src/test/java/com/stripe/android/link/account/FakeLinkAuth.kt

```diff
@@ -28,6 +29,7 @@ internal class FakeLinkAuth : LinkAuth {
                 consentAction = consentAction
             )
         )
+        yield()
```

Comment:
Why were these necessary?

Reply:
`signupTurbine.add()` is synchronous, so without `yield()` the `viewModelScope.launch {}` block in `onSignUpClick()` runs through without ever yielding to other coroutines, like that of `updateState {}`. If it never yields, we never get to see the intermediate state of `isSubmitting = true`.